presto-docs/src/main/sphinx/release/release-0.171.rst

```diff
@@ -0,0 +1,28 @@
+=============
+Release 0.171
+=============
+
+General Changes
+---------------
+
+* Fix planning failure when a query contains one ``DISTINCT`` and one or more aggregations functions.
```

Comment:
Fix planning regression for queries that compute a mix of distinct
  and non-distinct aggregations.